src/Symfony/Component/Mailer/composer.json

```diff
@@ -0,0 +1,44 @@
+{
+    "name": "symfony/mailer",
+    "type": "library",
+    "description": "Symfony Mailer Component",
+    "keywords": [],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Fabien Potencier",
+            "email": "fabien@symfony.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": "^7.1.3",
+        "psr/log": "~1.0",
+        "symfony/event-dispatcher": "^4.3",
+        "symfony/mime": "^4.3"
```

Comment:
`symfony/contracts` should be added here.

Reply:
It's not a requirement, only for the HTTP transports.